services/notifications/pkg/email/templates.go

```diff
@@ -72,18 +72,39 @@ You might still have access through your other groups or direct membership.`),
 
 Even though this membership has expired you still might have access through other shares and/or space memberships`),
 	}
+
+	ScienceMeshInviteTokenGenerated = MessageTemplate{
+		textTemplate: "templates/text/email.text.tmpl",
+		htmlTemplate: "templates/html/email.html.tmpl",
+		// ScienceMeshInviteTokenGenerated email template, Subject field (resolves directly)
+		Subject: l10n.Template(`ScienceMesh: {InitiatorName} wants to collaborate with you`),
+		// ScienceMeshInviteTokenGenerated email template, resolves via {{ .Greeting }}
+		Greeting: l10n.Template(`Hi,`),
+		// ScienceMeshInviteTokenGenerated email template, resolves via {{ .MessageBody }}
+		MessageBody: l10n.Template(`{ShareSharer} ({ShareSharerMail}) wants to start sharing OCM resources with you.
```

Comment:
```suggestion
		MessageBody: l10n.Template(`{ShareSharer} ({ShareSharerMail}) wants to start sharing collaboration resources with you.
```
This is user facing, IMHO, no one understands OCM...